matlab/@amimodel/generateC.m

```diff
@@ -113,11 +124,8 @@ function generateC(this)
 fprintf(fid,'\n');
 fprintf(fid,'\n');
 
-fprintf(fid,'#define pi M_PI\n');
-fprintf(fid,'\n');
 fprintf(fid,'#ifdef __cplusplus\n#define EXTERNC extern "C"\n#else\n#define EXTERNC\n#endif\n');
```

Comment:
Not strictly part of this PR, but we should remove that. Does not make sense since everything is C++ now.